### DISCUSSION TITLE
Searching for itens *not marked* as Favorite

### DISCUSSION THREAD

**Discussion by lopelisoli** [photoprism/photoprism#3296] [General]
Hi, I have a question about the behavior of the `favorite:` search flag. I went to [https://demo.photoprism.app](https://demo.photoprism.app) and performed the following steps:

1) I picked a picture and marked it as a Favorite (using the 'heart' icon).

2) In the search bar, I typed `favorite:true`. As expected, the image I marked as a Favorite is shown, while most others (not marked) are hidden.

3) Now I changed the search bar to `favorite:false`. Instead of seeing only images **not marked** Favorite, PhotoPrism seems to **show all images** - effectively ignoring the `favorite:false` flag.

Is this behavior expected? If so, how can I search for images not marked as Favorite?

**Comment by graciousgrey** [comment]
At the moment it is only possible to search for favorite:yes or favorite:true :point_right: https://docs.photoprism.app/user-guide/search/filters/#filter-reference

We do have an open feature request for a not operator, just no resources yet to implement it.

**Reply by lopelisoli** [reply]
Thank you for the clarification, @graciousgrey! I guess I was expecting `favorite:false` to be similar to `private:false` - which **works as expected**, even without the _not_ operator.

**Reply by lastzero** [reply]
Probably because it defaults to `public:true` in this special case.

**Comment by lopelisoli** [comment]
Ah, that makes sense! To check, I went back to [https://demo.photoprism.app](https://demo.photoprism.app) and tried to search for `private:foo` - and sure enough, the results were again equivalent to `public:true.`

Thanks for the explanation @lastzero. I'll wait for the NOT operator.
